embedding-reports/host-the-report-engine-remotely/rest-api-reference/json-entities/aithread.md

```diff
@@ -0,0 +1,51 @@
+---
+title: AiThread
+page_title: AiThread JSON Entity at a Glance
+description: "Learn more about the AiThread JSON Entity in Telerik Reporting REST Service and the type and meaning of each field."
+slug: telerikreporting/using-reports-in-applications/host-the-report-engine-remotely/telerik-reporting-rest-services/rest-api-reference/json-entities/aithread
+tags: aithread
+published: True
+position: 0
+---
+
+<style>
+table th:first-of-type {
```

Comment:
Are you sure these styles are correct? The table seems broken even on full screen on my monitor. Reducing the browser size makes it unreadable.
![table](image.png)